### DISCUSSION TITLE
Ability to rename sources in the sources list

### DISCUSSION THREAD

**Discussion by worldlydev** [wstyres/Zebra#1207] [Ideas]
https://www.reddit.com/r/jailbreak/comments/geqv8p/request_the_ability_to_nickname_repos_in_zebra_to/?utm_source=share&utm_medium=ios_app&utm_name=iossmf

Perhaps in the edit view like this (quick mock-up):
![9C6647A4-B86C-4CEB-AE5B-A27427B18413](https://user-images.githubusercontent.com/51668635/81382403-9eeb1e80-910e-11ea-8ed3-ff0106d7b164.png)
Or a button here:
![F9CD3FBE-1553-4956-B607-798889BFB943](https://user-images.githubusercontent.com/51668635/81382486-c2ae6480-910e-11ea-9fc8-6405caa789a9.jpeg)

**Comment by eve0415** [comment]
But I think that'll be reset when you "clear sources cache" or "remove the database because something went wrong" or "when you update Zebra" :(
